prow/monitoring/prometheus_operator_deployment.yaml

```diff
@@ -10,11 +10,11 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      k8s-app: prometheus-operator
+      app: prometheus-operator
   template:
     metadata:
       labels:
-        k8s-app: prometheus-operator
+        app: prometheus-operator
```

Comment:
Why is this changing here? To match a service monitor?

Reply:
Yeah, it was bugged before